### PR TITLE
Add XInput support for xbox controller Guide button

### PR DESCRIPTION
**Pull request by jpmikkers**

### CODE REVIEW

File: src/Vortice.XInput/XInput13.cs [hunk 1]
```diff
@@ -43,7 +43,7 @@ int IXInput.XInputGetAudioDeviceIds(int dwUserIndex, IntPtr renderDeviceId, IntP
             throw new NotSupportedException("XInputGetAudioDeviceIds is not supported on XInput1.3");
         }
 
-        [DllImport("xinput1_3.dll", CallingConvention = CallingConvention.StdCall)]
+        [DllImport("xinput1_3.dll", EntryPoint = "#100", CallingConvention = CallingConvention.StdCall)]
```
Comment (amerkoleci) [inline src/Vortice.XInput/XInput13.cs:46]: Why this change?

File: src/Vortice.XInput/XInput14.cs [hunk 2]
```diff
@@ -43,7 +43,7 @@ int IXInput.XInputGetAudioDeviceIds(int dwUserIndex, IntPtr renderDeviceId, IntP
             return XInputGetAudioDeviceIds(dwUserIndex, renderDeviceId, renderCount, captureDeviceId, captureCount);
         }
 
-        [DllImport("xinput1_4.dll", CallingConvention = CallingConvention.StdCall)]
+        [DllImport("xinput1_4.dll", EntryPoint = "#100", CallingConvention = CallingConvention.StdCall)]
```
Comment (amerkoleci) [inline src/Vortice.XInput/XInput14.cs:46]: Same as above

File: src/Vortice.XInput/XInput13.cs (hunk 1 above)
Comment (jpmikkers) [reply src/Vortice.XInput/XInput13.cs:46]: The regular entrypoint doesn't pass along the Guide flag, this one does.

Comment (jpmikkers) [reply src/Vortice.XInput/XInput13.cs:46]: Would you prefer a property on XInput, e.g. "AllowUnofficialApi" that would toggle the use of these?

**Comment by amerkoleci** [comment]
Is this part of Windows11 SDK? If so we need to avoid breaking backward compatibility and make sure it still works?

**Comment by jpmikkers** [comment]
#100 is an unofficial entrypoint, so I can understand you would not want to use these per default.  Perhaps a cleaner solution would be an alternative method in IXInput, XInputGetStateUnofficial ?

**Comment by amerkoleci** [comment]
> #100 is an unofficial entrypoint, so I can understand you would not want to use these per default. Perhaps a cleaner solution would be an alternative method in IXInput, XInputGetStateUnofficial ?

Yes, but then will need to find way to handle the implementation
